recipe/build.sh

```diff
@@ -1,19 +1,10 @@
-if [ ! -e tzdb-$PKG_VERSION.tar.lz ]; then
-    wget https://data.iana.org/time-zones/releases/tzdb-${PKG_VERSION}.tar.lz
-fi
-lzip -d tzdb-$PKG_VERSION.tar.lz
+lzip -d tzdb-${PKG_VERSION}.tar.lz
+tar xvf tzdb-${PKG_VERSION}.tar --strip 1
 
-tar xvf tzdb-$PKG_VERSION.tar --strip 1
-
-mkdir dest
 make -e \
-  cc=$CC \
-  DESTDIR=dest \
-  TOPDIR=$PREFIX \
-  USRDIR=. \
+  cc=${CC} \
+  TOPDIR=${PREFIX} \
+  USRDIR="" \
   install
 
-prefix_num_dirs=$(echo $PREFIX | grep -o / | wc -l)
-tar cf - -C dest . | tar xf - -C $PREFIX --strip $(( prefix_num_dirs + 1 ))
```

Comment:
This operation was leaking part of the build image into the package.
I guess that setting `USRDIR=""` has the same effect and fixes the leak.